File: pset1/mario/mario.c
#include<stdio.h>
#include<cs50.h>

int main(void)
{
    int n;

    do
    {

        n = get_int("length:\n");

    }

    while (n < 1 || n > 8);

    for (int a = 0; a < n; a++)
    {

        for (int b = n - 1; b > a; b--)
        {
            printf(" ");
        };
        for (int c = 0; c <= a; c++)
        {
            printf("#");
        };

        printf("  ");

        for (int c = 0; c <= a; c++)
        {
            printf("#");
        };

        printf("\n");

    };

}

File: pset6/mario/more/mario.py
from cs50 import get_int
while True:
    h = get_int("Height: ")
    if h > 0 and h <= 8:
        break

for i in range(h):
    j = h - 1
    k = 0
    l = 0
    while j > i:
        print(" ", end="")
        j -= 1

    while k <= i:
        print("#", end="")
        k += 1

    print("  ", end="")

    while l <= i:
        print("#", end="")
        l += 1

    print("")